config/install/user.role.az_content_editor.yml

```diff
@@ -7,6 +7,7 @@ weight: 2
 is_admin: false
 permissions:
   - 'access administration pages'
+  - 'access coffee'
```

Comment:
The ticket says this should be allowed only for administrators. If new discussions warrant the need for this module to be accessible by content editors/admins, it should only be added to the content editor role (not the content administrator role)